modules/react-components/src/components/tab/resource-tab/resource-tab.tsx

```diff
@@ -123,60 +127,60 @@ export const ResourceTab: FunctionComponent<ResourceTabPropsInterface> & Resourc
         setActiveIndex(activeIndex);
     };
 
-    if (isLoading === false) {
+    if (isLoading) {
         return (
-            <Tab
-                onTabChange={ (e: MouseEvent<HTMLDivElement>, data: TabProps) => {
-                    handleTabChange(e, data.activeIndex);
-                    onTabChange && typeof onTabChange === "function" && onTabChange(e, data);
-                } }
-                className={ classes }
-                menu={ {
-                    attached,
-                    pointing,
-                    secondary
-                } }
-                panes={ panes }
-                activeIndex={ activeIndex }
-                data-componentid={ componentId }
-                data-testid={ testId }
-                { ...rest }
-            />
+            <>
+                <Card.Group style={ { marginBottom: "3rem" } }>
```

Comment:
Better to avoid using inline css.

Reply:
Will be adding style classes for all placeholders in a future improvement PR.

Reply:
In that case, shall we add a TODO comment in the codebase mentioning that?

Reply:
Added TODO - https://github.com/wso2/identity-apps/pull/3629/commits/0d03052e55dd9150fdcb256b11f53ac2613360c5